src/main/java/org/kiwiproject/dropwizard/util/config/JacksonConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.kiwiproject.dropwizard.util.config;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class JacksonConfig {
```

Comment:
Do we need this to be externally configurable e.g. from a Dropwizard `config.yml`?

Reply:
Yes

Reply:
And you were about to tell me that I always forget that it works fine since Lombok's `@Builder` annotation generates an all-args constructor (private or package-private) that is annotated with `@ConstructorProperties` ???